### PR TITLE
Update FileSystemInfo.xml

### PR DESCRIPTION
**Pull request by sabarish98**
## Summary

Changed summary for FileSystemInfo.LinkTarget to provide better clarity on the property.

Fixes dotnet/runtime#62166

### CODE REVIEW

File: xml/System.IO/FileSystemInfo.xml [hunk 1]
```diff
@@ -1318,8 +1318,7 @@ The `Extension` property returns the <xref:System.IO.FileSystemInfo> extension,
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>If this <see cref="T:System.IO.FileSystemInfo" /> instance represents a link, returns the link target's path.
-            If a link does not exist in <see cref="P:System.IO.FileSystemInfo.FullName" />, or this instance does not represent a link, returns <see langword="null" />.</summary>
+        <summary>Gets the target path of the current file or directory, if the <see cref="T:System.IO.FileSystemInfo" /> instance represents a link or a link exists at <see cref="P:System.IO.FileSystemInfo.FullName" />. Otherwise, returns <see langword="null" />.</summary>
         <value>To be added.</value>
```
Comment (gewarren) [inline xml/System.IO/FileSystemInfo.xml:1322]:
```suggestion
        <value>The target path of the symbolic link located in <see cref="P:System.IO.FileSystemInfo.FullName" />, or <see langword="null" /> if this <see cref="T:System.IO.FileSystemInfo" /> instance doesn't represent a link.</value>
```

**Comment by sabarish98** [comment]
> If the instance represents a link, then wouldn't a link have to exist at FullName? I think these are one and the same?

@gewarren : Yes, that's correct. But when we work with file paths dynamically, I feel we will use FullName to find the full path since we don't know with what value the instance was created with. So, I feel, it's valid to tell that.

File: xml/System.IO/FileSystemInfo.xml (hunk 1 above)
Comment (sabarish98) [reply xml/System.IO/FileSystemInfo.xml]: Added the value

File: xml/System.IO/FileSystemInfo.xml [hunk 2]
```diff
@@ -1318,8 +1318,7 @@ The `Extension` property returns the <xref:System.IO.FileSystemInfo> extension,
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>If this <see cref="T:System.IO.FileSystemInfo" /> instance represents a link, returns the link target's path.
-            If a link does not exist in <see cref="P:System.IO.FileSystemInfo.FullName" />, or this instance does not represent a link, returns <see langword="null" />.</summary>
+        <summary>Gets the target path of the current file or directory, if the <see cref="T:System.IO.FileSystemInfo" /> instance represents a link or a link exists at <see cref="P:System.IO.FileSystemInfo.FullName" />. Otherwise, returns <see langword="null" />.</summary>
```
[the remark this answers is not in the document]
Comment (sabarish98) [reply xml/System.IO/FileSystemInfo.xml]: Incorporated the changes as suggested : )

**Review by gewarren — APPROVED** [review]
Thanks, LGTM!

**Comment by sabarish98** [comment]
@gewarren : What more do we need to do to get this change merged. Do we need another review?

**Comment by gewarren** [comment]
> @gewarren : What more do we need to do to get this change merged. Do we need another review?

I'd like to get a review form @Jozkee before we merge.

File: xml/System.IO/FileSystemInfo.xml [hunk 3]
```diff
@@ -1318,9 +1318,8 @@ The `Extension` property returns the <xref:System.IO.FileSystemInfo> extension,
         <ReturnType>System.String</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>If this <see cref="T:System.IO.FileSystemInfo" /> instance represents a link, returns the link target's path.
-            If a link does not exist in <see cref="P:System.IO.FileSystemInfo.FullName" />, or this instance does not represent a link, returns <see langword="null" />.</summary>
-        <value>To be added.</value>
+        <summary>Gets the target path of the symbolic link located in <see cref="P:System.IO.FileSystemInfo.FullName" />, or <see langword="null" /> if this <see cref="T:System.IO.FileSystemInfo" /> instance doesn't represent a link.</summary>
```
Comment (Jozkee) [inline xml/System.IO/FileSystemInfo.xml:1321]:
```suggestion
        <summary>Gets the target path of the link located in <see cref="P:System.IO.FileSystemInfo.FullName" />, or <see langword="null" /> if this <see cref="T:System.IO.FileSystemInfo" /> instance doesn't represent a link.</summary>
```
Do not specify symbolic link as this API also works with junctions which are another kind of link only supported by NTFS.

File: xml/System.IO/FileSystemInfo.xml [hunk 4]
```diff
@@ -1318,9 +1318,8 @@ The `Extension` property returns the <xref:System.IO.FileSystemInfo> extension,
...
+        <value>The target path of the symbolic link located in <see cref="P:System.IO.FileSystemInfo.FullName" />, or <see langword="null" /> if this <see cref="T:System.IO.FileSystemInfo" /> instance doesn't represent a link.</value>
```
Comment (Jozkee) [inline xml/System.IO/FileSystemInfo.xml:1322]:
```suggestion
        <value>The target path of the link located in <see cref="P:System.IO.FileSystemInfo.FullName" />, or <see langword="null" /> if this <see cref="T:System.IO.FileSystemInfo" /> instance doesn't represent a link.</value>
```

File: xml/System.IO/FileSystemInfo.xml (hunk 3 above)
Comment (sabarish98) [reply xml/System.IO/FileSystemInfo.xml]: Noted @Jozkee. Made the change

File: xml/System.IO/FileSystemInfo.xml (hunk 4 above)
Comment (sabarish98) [reply xml/System.IO/FileSystemInfo.xml]: Changed this as well

**Pull request merged**